package.json

```diff
@@ -26,8 +26,6 @@
     "homepage": "https://github.com/eventbrite/eventbrite-sdk-javascript#readme",
     "license": "MIT",
     "scripts": {
-        "check:size": "bundlesize",
-        "precheck:size": "yarn gulp build:dist:min",
         "check:static": "npm-run-all --parallel lint tsc",
```

Comment:
Is the plan to remove this once you get Typescript working in Jest? As of right now `tsc` no longer exists...

Reply:
Oh yes, that was my intent, I thought I removed it. I think I put it back since it was not in my checklist. 😄